index.html

```diff
@@ -641,26 +641,69 @@ <h2>Accessibility Considerations</h2>
+                The usual scenarios where digital signatures are included in MiniApps are, but are not limited to:
+            </p> 
             <ul>  
-                <li>A digital signature (with a valid certificate) of the author that ensures the origin of the MiniApp, which gives a recipient strong reason to believe that this package was created by a known sender (authentication) and that the message was not altered in transit (integrity). Thus, an end-user or a hosting platform can decide whether to install the [=MiniApp package=] depending on the status and quality of the author (e.g., credits, blocklist, certifications, etc.).</li>
-                <li>Distributors’ signatures (for example, official marketplace signatures), which are used to ensure the trustworthiness of the delivery channel. This kind of signature protects end-users from tampering activities and ensures a healthy ecosystem of applications. </li>
+                <li>
+                    A [=digital signature=] (with a valid certificate) of the author that ensures the origin of the MiniApp, which gives a recipient strong reason to believe that this package was created by a known sender (authentication) and that the message was not altered in transit (integrity). Thus, an end-user or a hosting platform can decide whether to install the [=MiniApp package=] depending on the status and quality of the author (e.g., credits, blocklist, certifications, etc.).
+                </li>
+                <li>
+                    Distributors’ signatures (for example, official marketplace signatures), which are used to ensure the trustworthiness of the delivery channel. This kind of signature protects end-users from tampering activities and provides a healthy ecosystem of applications.
+                </li>
             </ul>
-            <p>No specific digital signature mechanism, cryptographic technology, or algorithm is recommended in this specification. MiniApp vendors MAY or MAY NOT implement digital signature mechanisms, according to their use cases. When implemented, the <a href="#sec-miniapp-digital-signature-requirements"></a> shall be followed.</p>
-
-            <p>Some examples of technologies and encryption methods are introduced in the <a href="#sec-signature-schemes"></a>.</p>
-
-        </section>
-        <section>
-            <h3>Content Confidentiality</h3>
-            <p>This specification does not recommend any encryption mechanism for the [=MiniApp package=] to protect its confidentiality. However, it doesn't preclude an implementation from deploying some encryption mechanism for special purposes.</p>
+            <p>
+                No specific [=digital signature=] mechanism, cryptographic technology, or algorithm is recommended in this specification. MiniApp vendors MAY or MAY NOT implement digital signature mechanisms, according to their use cases. When implemented, the <a href="#sec-miniapp-digital-signature-requirements"></a> shall be followed.
```

Comment:
some nits:

* I think the RFC 2119 MAY already means "MAY or MAY NOT"
* "shall" is also an RFC 2119 keyword, so you may (no pun intended) want to make it all capitals

Reply:
Well spotted. Thanks @xfq , solved with 14218c8228693ff9b62f762fcf3f643f20b70ad0